### DISCUSSION TITLE
Incomplete Documentation: How to Install

### DISCUSSION THREAD

**Discussion by CorruptComputer** [louislam/dockge#630] [Ask for Help]
### ⚠️ Please verify that this bug has NOT been reported before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/dockge/security/policy)

### Description

Something is missing from the how to install documentation, I've followed it exactly on a brand new install of CentOS using podman and it does not work.

https://github.com/louislam/dockge?tab=readme-ov-file#-how-to-install

### 👟 Reproduction steps

1. Setup a brand new install of CentOS Stream 9.
2. Follow the how to install documentation exactly, do not make any assumptions about anything not listed as a requirement.
3. The installation will fail, because something is missing.

### 👀 Expected behavior

You should be able to install the application successfully.

### 😓 Actual Behavior

```bash
[root@echidna ~]# mkdir -p /opt/stacks /opt/dockge
[root@echidna ~]# cd /opt/dockge
[root@echidna dockge]# curl https://raw.githubusercontent.com/louislam/dockge/master/compose.yaml --output compose.yaml
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   775  100   775    0     0   3974      0 --:--:-- --:--:-- --:--:--  3994
[root@echidna dockge]# docker compose up -d
Emulate Docker CLI using podman. Create /etc/containers/nodocker to quiet msg.
Error: looking up compose provider failed
7 errors occurred:
	* exec: "docker-compose": executable file not found in $PATH
	* exec: "/root/.docker/cli-plugins/docker-compose": stat /root/.docker/cli-plugins/docker-compose: no such file or directory
	* exec: "/usr/local/lib/docker/cli-plugins/docker-compose": stat /usr/local/lib/docker/cli-plugins/docker-compose: no such file or directory
	* exec: "/usr/local/libexec/docker/cli-plugins/docker-compose": stat /usr/local/libexec/docker/cli-plugins/docker-compose: no such file or directory
	* exec: "/usr/lib/docker/cli-plugins/docker-compose": stat /usr/lib/docker/cli-plugins/docker-compose: no such file or directory
	* exec: "/usr/libexec/docker/cli-plugins/docker-compose": stat /usr/libexec/docker/cli-plugins/docker-compose: no such file or directory
	* exec: "podman-compose": executable file not found in $PATH
```

### Dockge Version

n/a

### 💻 Operating System and Arch

CentOS Stream 9

### 🌐 Browser

n/a

### 🐋 Docker Version

Podman

### 🟩 NodeJS Version

n/a

**Comment by Jamie-** [comment]
Looks like when you installed Docker, you've missed the compose plugin. Following the official Docker install docs for CentOS you should have installed `docker-compose-plugin`.
Try installing it with `sudo yum install docker-compose-plugin` and trying again?

**Reply by CorruptComputer** [reply]
I'm not using docker, I am using podman. But I've found that just using Cockpit's Podman UI works well enough for my use case so I'm not running this anymore.
